desktop-src/WES/defining-events.md

```diff
@@ -12,99 +12,91 @@ Providers must define all the events that they write. To define an event, use th
 
 You should set the **symbol** attribute to specify the symbolic constant that the compiler generates to identify the event's event descriptor—you use the event descriptor when writing the event. If you do not specify the symbol, the compiler will generate a name for you.
 
 The **message** attribute contains the localized message string that is displayed with the event. To include event-specific data in the string, add insertion strings to the message text. For example, to include the third data item in the template, include %3.
 
-The following example shows how to define events.
-
+The following example shows a complete manifest that defines events.
 
 ```XML
 <instrumentationManifest
-    xmlns="http://schemas.microsoft.com/win/2004/08/events" 
-    xmlns:win="https://manifests.microsoft.com/win/2004/08/windows/events"
-    xmlns:xs="https://www.w3.org/2001/XMLSchema"    
+    xmlns="http://schemas.microsoft.com/win/2004/08/events"
+    xmlns:win="http://manifests.microsoft.com/win/2004/08/windows/events"
+    xmlns:xs="https://www.w3.org/2001/XMLSchema"
     >
-
     <instrumentation>
         <events>
-            <provider name="Microsoft-Windows-SampleProvider" 
-                guid="{1db28f2e-8f80-4027-8c5a-a11f7f10f62d}" 
-                symbol="PROVIDER_GUID" 
-                resourceFileName="<path to the exe or dll that contains the metadata resources>" 
+            <provider name="Microsoft-Windows-SampleProvider"
+                guid="{1db28f2e-8f80-4027-8c5a-a11f7f10f62d}"
+                symbol="PROVIDER_GUID"
+                resourceFileName="<path to the exe or dll that contains the metadata resources>"
                 messageFileName="<path to the exe or dll that contains the string resources>"
                 message="$(string.Provider.Name)">
 
                 <channels>
-                    <channel chid="c1" 
-                             name="Microsoft-Windows-SampleProvider/Admin"
-                             type="Admin"
-                             enabled="true">
-
-                    <channel chid="c2" 
-                             name="Microsoft-Windows-SampleProvider/Operational"
-                             type="Operational"
-                             enabled="true">
+                    <channel chid="c1"
+                             name="Microsoft-Windows-BaseProvider/Admin"
+                             symbol="CHANNEL_BASEPROVIDER_ADMIN"
+                             type="Admin"/>
                 </channels>
 
+                <levels>
+                    <level name="NotValid"
```

Comment:
What is this for? I'm confused.

Reply:
What do you mean? This is here because it came from the defining-severity-levels example.